.ncurc.yaml

```diff
@@ -4,5 +4,7 @@ reject: [
     # Issue is tracked here: https://github.com/mojaloop/project/issues/3616
     "get-port",
     # glob >= 11 requires node >= 20
-    "glob"
+    "glob",
+    # v2 introduces breaking changes. Address this update later.
+    "@mojaloop/ml-schema-transformer-lib"
```

Comment:
We need to take care of this before merging

Reply:
Done.